src/Forms/GroupForm/GroupForm.jsx

```diff
@@ -159,6 +164,14 @@ const GroupForm = ({
           }}
           value={name}
         />
+        <TextInput
+          required
+          placeholder={t("global.description")}
```

Comment:
can you use the `groupDescription` translation?

Reply:
done